docs/ABOUT.md

```diff
@@ -2,8 +2,8 @@
 
 "Racket (formerly named PLT Scheme) is a general purpose, multi-paradigm programming language in the Lisp/Scheme family. One of its design goals is to serve as a platform for language creation, design, and implementation. The language is used in a variety of contexts such as scripting, general-purpose programming, computer science education, and research." - [Wikipedia](https://en.wikipedia.org/wiki/Racket_(programming_language) "Wikipedia page on racket")
 
-You can learn the basics of Racket through a coule of books.  Start by checking out [How to Design Programs](https://htdp.org "How to Design Programs website") or the slightly less formal but enjoyable [Realm of Racket](http://www.realmofracket.com/ "Realm of Racket website").
+You can learn the basics of Racket through a few different books.  Start by checking out [*How to Design Programs*](https://htdp.org "How to Design Programs website"), the slightly less formal but enjoyable [*Realm of Racket*](http://www.realmofracket.com/ "Realm of Racket website"), or [*Racket Programming the Fun Way*](https://nostarch.com/racket-programming-fun-way "Racket Programming the Fun Way store page at No Starch Press").
```

Comment:
Ooof.  I hate having to do this to you, but our  [exercisim markdown spec](https://exercism.org/docs/building/markdown/markdown) asks that links be _reference links_, defined at the bottom of the file.

Reply:
no problem, was just going by what was already in the file -- I'll put a fix in momentarily